pkg/config/loader/loader.go

```diff
@@ -412,6 +412,34 @@ func GetServerConfigFromConfigfile(dc *database.Config, cf *server.ServerConfigF
 		})
 	}
 
+	if cf.Auth.Custom.Enabled {
+		if cf.Auth.Custom.ClientID == "" {
+			return nil, nil, fmt.Errorf("custom client id is required")
+		}
+
+		if cf.Auth.Custom.ClientSecret == "" {
+			return nil, nil, fmt.Errorf("custom client secret is required")
+		}
+
+		if cf.Auth.Custom.AuthorizationURL == "" {
```

Comment:
Some validation via `url.Parse` on the `AuthorizationURL` and `AccessTokenURL` would be useful here